test/scripts/build-alibi-explainer.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to build an image from our argo workflow
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REGISTRY="${GCP_REGISTRY}"
+PROJECT="${GCP_PROJECT}"
+VERSION=$(git describe --tags --always --dirty)
+
+echo "Activating service-account"
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+cd python
```

Comment:
Can we consolidate all the python images into a single script and pass the list of components as script argument ?

Reply:
yes, we can. But all image build will run one by one instead of in parallel, which will take more time to finish test.

Reply:
Ok, I misunderstood what you mean.
Had changed to one script.